test/parallel/test-tls-basic-validations.js

```diff
@@ -56,3 +56,17 @@ assert.throws(() => tls.createSecurePair({}),
   assert(buffer.equals(Buffer.from('abcd')));
   assert(out.NPNProtocols.equals(Buffer.from('efgh')));
 }
+
+{
+  const buffer = new Uint8Array([...Buffer.from('abcd')]);
```

Comment:
@lpinca, heh I was mistaken. In that case, yeah there's no reason to use spread syntax...